src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRelocateResource.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2023 Crown Copyright (Health Education England)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.nhs.hee.tis.trainee.forms.api;
+
+import com.amazonaws.xray.spring.aop.XRayEnabled;
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.nhs.hee.tis.trainee.forms.api.util.HeaderUtil;
+import uk.nhs.hee.tis.trainee.forms.service.FormRelocateService;
+import uk.nhs.hee.tis.trainee.forms.service.exception.ApplicationException;
+
+@Slf4j
+@RestController
+@RequestMapping("/api")
+@XRayEnabled
+public class FormRelocateResource {
+
+  private final FormRelocateService service;
+
+  public FormRelocateResource(FormRelocateService service) {
+    this.service = service;
+  }
+
+  /**
+   * POST  /form-relocate : Relocate a form (attach a form from one trainee to another).
+   *
+   * @param formId  The ID of the form that need to relocate
+   * @param targetTrainee  The TraineeTisId of the target trainee
+   * @return the status of relocation
+   */
+  @PostMapping("/form-relocate/{formId}")
```

Comment:
I wonder whether a PATCH mapping might be more appropriate here (assuming that can be integrated with slack, of course!)